index.bs

```diff
@@ -9118,11 +9119,44 @@ storage.PartitionKey = {
 
 The <code>storage.PartitionKey</code> type represents a [=storage partition key=].
 
-A [=remote end=] has a [=/set=] of <dfn>storage partition key attributes</dfn> which contains zero or more implementation-defined string values.
-
-A [=remote end=] has a [=/map=] of <dfn>default values for storage partition key attributes</dfn> which contains zero or more implementation-defined entries. Each key must be a member of the [=storage partition key attributes=] which the [=remote end=] does not require as input to [=storage.getCookies=], [=storage.setCookie=], and [=storage.deleteCookies=]; the corresponding value is a the value which the [=remote end=] uses when one is not provided.
-
-A [=remote end=] has a [=/list=] of <dfn>required partition key attributes</dfn> which contains zero or more implementation-defined entries.
+The following <dfn>table of standard storage partition key attributes</dfn>
+enumerates attributes with well-known meanings which a [=remote end=] may
+choose to support. An implementation may define additional [=extension storage
+partition key attributes=].
+
+<table>
+  <tr>
+    <th>Attribute
+    <th>Definition
+  <tr>
+    <td>"<code>userContext</code>"
+    <td>
+      A user context id
+
+      Issue(w3c/webdriver-bidi#570): User contexts are in the process of being
+      specified.
+  <tr>
+    <td>"<code>sourceOrigin"</code>
+    <td>The [=origin=] from which the storage partition is to be accessed
+</table>
+
+[=Remote ends=] may support any number of <dfn>extension storage partition key
+attributes</dfn>. In order to avoid conflicts with other implementations, these
+attributes must begin with a unique identifier for the vendor and user-agent
+followed by U+003A (:).
+
+A [=remote end=] has a [=/map=] of <dfn>default values for storage partition
+key attributes</dfn> which contains zero or more implementation-defined
+entries. Each key must be a member of the [=table of standard storage partition
+key attributes=] or an [=extension storage partition key attribute=].
+
+Note: Default values are used by [=storage.getCookies=], [=storage.setCookie=],
+and [=storage.deleteCookies=] when the [=local end=] does not specify a value.
+
+A [=remote end=] has a [=/list=] of <dfn>required partition key
```

Comment:
Maybe also rephrase this more like the one above.

Reply:
Addressed via commit 9eebfc1.